src/core/TChem_AerosolModelData.cpp

```diff
@@ -30,7 +30,7 @@ void AerosolModelData::setGasParameters(const KineticModelData& kmd){
   // maps from kmd has order of gas species.
   gas_sp_name_idx_ = kmd.species_indx_;
   is_gas_parameters_set_=true;
-  printf("Setting gas parameters... nSpec_gas_ %d  nConstSpec_gas_%d \n", nSpec_gas_, nConstSpec_gas_);
+  printf("[AerosolModelData::setGasParameters] Setting gas parameters... nSpec_gas_ %d  nConstSpec_gas_%d \n", nSpec_gas_, nConstSpec_gas_);
```

Comment:
Thanks for improving this print statement. I would like to save all these messages to a file. I have created issue #45.